src/internal_events/loki.rs

```diff
@@ -1,13 +1,32 @@
+use super::prelude::{error_stage, error_type};
 use metrics::counter;
 use vector_core::internal_event::InternalEvent;
 
 #[derive(Debug)]
-pub struct LokiEventUnlabeled;
+pub struct LokiEventUnlabeledError;
+
+impl InternalEvent for LokiEventUnlabeledError {
+    fn emit_logs(&self) {
+        error!(
+            message = "Unlabeled event, setting defaults.",
+            error_code = "unlabeled",
+            error_type = error_type::CONDITION_FAILED,
+            stage = error_stage::PROCESSING,
+        );
+    }
 
-impl InternalEvent for LokiEventUnlabeled {
     fn emit_metrics(&self) {
-        counter!("processing_errors_total", 1,
-                "error_type" => "unlabeled_event");
+        counter!(
+            "component_errors_total", 1,
```

Comment:
This is starting me wondering if we shouldn't have a separate metric name for warnings, but that can happen later.

Reply:
I totally agree with you regarding that. We have a tendency to mix stuff 🤔